models/modules/resnet_architecture/mobile_resnet_generator.py

```diff
@@ -82,6 +84,10 @@ def __init__(self, input_nc, output_nc, ngf, norm_layer=nn.InstanceNorm2d,
 
         mult = 2 ** n_downsampling
 
+        if use_spatial:
+            # model += [nn.Conv2d(ngf * mult, 4, kernel_size=3, padding=1, bias=use_bias)]
+            model += [SpatialTransform(in_features = ngf * mult * 64 * 64)]
```

Comment:
Yeah, here we've already discussed that Conv2d()->AdaptiveAvgPool2d(1)->1x1x6 tensor is better (common way to  treat params like "one param == one feature channel").